src/db/dbMigrator/dbMigrator.ts

```diff
@@ -10,13 +11,15 @@ const logger = new Logger('DBMigrator')
 const migrateSchema = async (params: { schema?: string; migrationsFolder?: string } = {}): Promise<void> => {
   const { schema = Schemata.PUBLIC, migrationsFolder = __dirname } = params
 
-  if (schema !== Schemata.PUBLIC) {
-    await DB.none(`CREATE SCHEMA IF NOT EXISTS ${schema}`)
-  }
+  if (fs.existsSync(migrationsFolder)) {
```

Comment:
Should we avoid nested ifs?
`if (!fs.existsSync(migrationsFolder)) return`

Reply:
yes, see my last commit